app/code/Magento/Checkout/Block/Cart/Grid.php

```diff
@@ -157,6 +157,16 @@ public function getItems()
         return $this->getItemsForGrid()->getItems();
     }
 
+    /**
+     * Return validation url for shopping cart update form
+     *
+     * @return string
+     */
+    public function getValidationUrl()
```

Comment:
new functions must have an explicit return type 
https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html
```suggestion
    public function getValidationUrl(): string
```